Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.54.0
+FROM rust:1.62.0
```

Comment:
I don't know if we want to do this, this is a library after all & we should be sure that it works with the "oldest possible"  version of Rust

wdyt?

Reply:
I think this is mandatory - this isn't due to a dependencies change, but the CI falling over. Can't merge if I don't bump this.

Reply:
Even master started breaking, I guess due to a dependency...? Anyway, let's bump it

Reply:
Should we also add `rust-version` to the Cargo.toml to enforce the MSRV?